hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/state/WrappedWritableKVState.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.spi.state;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Iterator;
+import java.util.Objects;
+
+/**
+ * An implementation of {@link WritableKVState} that delegates to another {@link WritableKVState} as
+ * though it were the backend data source. Modifications to this {@link WrappedWritableKVState} are
```

Comment:
Misplaced comment? This class doesn't seem to do any buffering; just looks like a convenient base class to facilitate subclasses that want to instrument a delegate.

Reply:
Correct @netopyr !